tests/common/dummy_data.rs

```diff
@@ -160,31 +167,31 @@ pub async fn add_project_alpha(test_env: &TestEnvironment) -> (Project, Version)
     (project, version)
 }
 
-pub async fn add_project_beta(test_env: &TestEnvironment) -> (Project, Version) {
+pub async fn add_public_dummy_project(
```

Comment:
I think the one part I disagree with is having `add_public_dummy_project` as a function to be used in tests. Im all for a helper function for easy creation of projects (like you are doing for `TestEnvironment`) , but for dummy data specifically im not sure.

 Mostly because the speedup changes involve putting the dummy data into a template db, meaning that you generate the dummy data in the first test and never again, and it doesn't vary test to test (so it feels wrong to generate dummy data in tests).

We could instead do a `add_public_project` on `TestEnvironment` or something similar- what do you think

Reply:
To be clear I think having an easy create-project function is good but associating with the dummy data is what I disagree with

Reply:
Oh good point - the name felt weird anyway. I like the idea of moving to to `TestEnvironment`, will do.